File: C_Sharp_code/Example/WindowsFormsApp/VB_demo05/Form1.vb
Public Class Form1
    Enum MyColor
        Red = 1
        Blue = 2
        Yellow = 3
        '沒有指派的話會由索引0開始
    End Enum
    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        Dim c As MyColor = MyColor.Blue

        Console.WriteLine(c)
        Console.WriteLine(c.ToString()) '顯示列舉項目

    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo05/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo05
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        enum MyColor { Red,Blue,Yellow}//列舉不能instance
        private void button1_Click(object sender, EventArgs e)
        {
            MyColor c = MyColor.Yellow;
            int index = (int)c;//列舉項目的索引由0開始
            Console.WriteLine(index);
        }
    }
}
